api/peripheral_api/routes/Speaker.js

```diff
@@ -0,0 +1,79 @@
+const express = require('express');
+const axios = require('axios');
+const router = express.Router();
+const { speakerQueued } = require('../util/Speaker.js');
+const {
+  verifyToken,
+  checkIfTokenSent,
+} = require('../../util/token-verification');
+const {
+  OK,
+  BAD_REQUEST,
+  UNAUTHORIZED,
+  FORBIDDEN,
+  NOT_FOUND,
+} = require('../../util/constants').STATUS_CODES;
+const logger = require('../../util/logger');
+
+router.post('/stream', async (req, res) => {
```

Comment:
all of these functions should be expecting a token other wise anyone can play music on the speakers over the public internet